crates/typst/src/text/deco.rs

```diff
@@ -280,13 +280,19 @@ pub struct HighlightElem {
     /// The color to highlight the text with.
     ///
     /// ```example
-    /// This is #highlight(fill: blue)[with blue].
+    /// This is #highlight(fill: blue)[highlighted with blue].
     /// ```
     #[default(Color::from_u8(0xFF, 0xFD, 0x11, 0xA1).into())]
     pub fill: Paint,
 
     /// The highlight's border color. See the
     /// [rectangle's documentation]($rect.stroke) for more details.
+    ///
+    /// ```example
+    /// This is a #highlight(
+    ///   stroke: 2pt + orange.lighten(70%)
+    /// )[stroked highlighting].
```

Comment:
```suggestion
    /// )[stroked highlighting.]
```

I think the convention in English would be to include the dot within the highlighted text here. I may be wrong though, but I think that, generally, punctuation marks are moved inside quotes/styling (italics, bold, etc.).

Same for the comma in the next example.

Reply:
What about the example above? And the one above that one? There are 5 examples and all of them use the same pattern. 

I can agree with things like period and quotes, but highlighting? I think highlighting in general is a subjective thing, as I practically never use it in Typst or in real life. But others do, and they all probably highlight very differently. I would be very surprised if there are highlighting rules out there.

Besides, these people: frozolotl, LU Jialin have passed a code review, which means that maintainers (1–2) as well as contributors (3) are ok with the way things are right now.

Reply:
I think it's a bit too subjective to force a choice here, so we should just keep the example as is here imo.

Reply:
> What about the example above? And the one above that one? There are 5 examples and all of them use the same pattern.

I think their proposal was to change that wherever applicable, so those examples would be changed as well. But I agree it's not really necessary.

Reply:
That was just an introduction (with rhetorical questions) to the overall problem. It is logical that if changed in one place, then it should be changed in all places. I just noted that there are more instances of that same pattern than just 2 examples that I've added.

Reply:
I agree it doesn't matter. I just wanted to point it out just in case. But if all the other examples do it that way let's just keep it as is.